deploy/ray-operator/config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: ray-operator-system
+namespace: ray
```

Comment:
Why change this?

Reply:
The k8s node provider creates all the stuff(like serviceAccount) in namespace ray, for users are easy to use, just apply the k8s operator in the same namespace to reduce usage cost.